### ISSUE TITLE
Panel: without header leads to broken ARIA reference

### ISSUE THREAD

**Issue by timdoering** [primefaces/primefaces#10379]
### Describe the bug

If you use a p:panel and don't declare a header attribute you will get a broken aria reference ("aria-labelledby")

### Reproducer

Any use of just the panel will reproduce the issue:
`<p:panel>`

Using a header seems to fix it, but sometimes I don't need a header:
`<p:panel header="headertext">`

### Expected behavior

Aria labels should be fixed

### PrimeFaces edition

None

### PrimeFaces version

13.0.0

### JSF implementation

Mojarra

### JSF version

2.x

### Java version

11

**Comment by melloware** [comment]
I will fix this.

**Comment by melloware** [comment]
PR submitted please review

**Comment by melloware** [comment]
13.0.1-SNAPSHOT is available with your fix if you want to test: https://github.com/primefaces/primefaces#snapshot
